<Type Name="NotFiniteNumberException" FullName="System.NotFiniteNumberException">
  <TypeSignature Language="C#" Value="public class NotFiniteNumberException : ArithmeticException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit NotFiniteNumberException extends System.ArithmeticException" />
  <TypeSignature Language="DocId" Value="T:System.NotFiniteNumberException" />
  <TypeSignature Language="VB.NET" Value="Public Class NotFiniteNumberException&#xA;Inherits ArithmeticException" />
  <TypeSignature Language="C++ CLI" Value="public ref class NotFiniteNumberException : ArithmeticException" />
  <AssemblyInfo>
    <AssemblyName>mscorlib</AssemblyName>
    <AssemblyVersion>2.0.5.0</AssemblyVersion>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>netstandard</AssemblyName>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <AssemblyInfo>
    <AssemblyName>System.Runtime</AssemblyName>
    <AssemblyVersion>4.2.0.0</AssemblyVersion>
    <AssemblyVersion>4.2.1.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.ArithmeticException</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute>
      <AttributeName>System.Runtime.InteropServices.ComVisible(true)</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary><span data-ttu-id="568b7-101">浮動小数点値が正の無限大、負の無限大、または非数 (NaN) の場合にスローされる例外。</span><span class="sxs-lookup"><span data-stu-id="568b7-101">The exception that is thrown when a floating-point value is positive infinity, negative infinity, or Not-a-Number (NaN).</span></span></summary>
    <remarks>
      <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="568b7-102"><xref:System.NotFiniteNumberException> 浮動小数点演算で、無限大および Not 数の概念をサポートしていないプログラミング言語で使用できます。</span><span class="sxs-lookup"><span data-stu-id="568b7-102"><xref:System.NotFiniteNumberException> is available for programming languages that do not support the concepts of infinity and Not-a-Number in floating-point operations.</span></span>  
  
 <span data-ttu-id="568b7-103"><xref:System.NotFiniteNumberException> 値は 0x80131528 HRESULT COR_E_NOTFINITENUMBER を使用します。</span><span class="sxs-lookup"><span data-stu-id="568b7-103"><xref:System.NotFiniteNumberException> uses the HRESULT COR_E_NOTFINITENUMBER, which has the value 0x80131528.</span></span>  
  
 <span data-ttu-id="568b7-104">インスタンスの初期プロパティ値の一覧については<xref:System.NotFiniteNumberException>を参照してください、<xref:System.NotFiniteNumberException.%23ctor%2A>コンス トラクターです。</span><span class="sxs-lookup"><span data-stu-id="568b7-104">For a list of initial property values for an instance of <xref:System.NotFiniteNumberException>, see the <xref:System.NotFiniteNumberException.%23ctor%2A> constructors.</span></span>  
  
 ]]></format>
    </remarks>
    <altmember cref="T:System.Exception" />
    <altmember cref="T:System.Double" />
    <altmember cref="T:System.Single" />
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary><span data-ttu-id="568b7-105"><see cref="T:System.NotFiniteNumberException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="568b7-105">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class.</span></span></summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary><span data-ttu-id="568b7-106"><see cref="T:System.NotFiniteNumberException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="568b7-106">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="568b7-107">このコンス トラクター、 <xref:System.Exception.Message%2A> "、見つかった数がありませんは有限な数です"などに、エラーを説明するシステム提供のメッセージの新しいインスタンスのプロパティ。</span><span class="sxs-lookup"><span data-stu-id="568b7-107">This constructor initializes the <xref:System.Exception.Message%2A> property of the new instance to a system-supplied message that describes the error, such as "The number encountered was not a finite quantity."</span></span> <span data-ttu-id="568b7-108">このメッセージには、現在のシステムのカルチャが考慮されます。</span><span class="sxs-lookup"><span data-stu-id="568b7-108">This message takes into account the current system culture.</span></span>  
  
 <span data-ttu-id="568b7-109"><xref:System.NotFiniteNumberException> のインスタンスの初期プロパティ値を次の表に示します。</span><span class="sxs-lookup"><span data-stu-id="568b7-109">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="568b7-110">プロパティ</span><span class="sxs-lookup"><span data-stu-id="568b7-110">Property</span></span>|<span data-ttu-id="568b7-111">[値]</span><span class="sxs-lookup"><span data-stu-id="568b7-111">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="568b7-112">null 参照 (Visual Basic の場合は `Nothing`)。</span><span class="sxs-lookup"><span data-stu-id="568b7-112">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="568b7-113">ローカライズされたエラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="568b7-113">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException (double offendingNumber);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(float64 offendingNumber) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.Double)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (offendingNumber As Double)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException(double offendingNumber);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="offendingNumber" Type="System.Double" />
      </Parameters>
      <Docs>
        <param name="offendingNumber"><span data-ttu-id="568b7-114">例外の原因となった引数の値。</span><span class="sxs-lookup"><span data-stu-id="568b7-114">The value of the argument that caused the exception.</span></span></param>
        <summary><span data-ttu-id="568b7-115">無効な数値を使用して、<see cref="T:System.NotFiniteNumberException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="568b7-115">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with the invalid number.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="568b7-116">`offendingNumber`パラメーターは両方を指定する必要があります、<xref:System.Double>と数が無効です。</span><span class="sxs-lookup"><span data-stu-id="568b7-116">The `offendingNumber` parameter must be both a <xref:System.Double> and an invalid number.</span></span>  
  
 <span data-ttu-id="568b7-117"><xref:System.NotFiniteNumberException> のインスタンスの初期プロパティ値を次の表に示します。</span><span class="sxs-lookup"><span data-stu-id="568b7-117">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="568b7-118">プロパティ</span><span class="sxs-lookup"><span data-stu-id="568b7-118">Property</span></span>|<span data-ttu-id="568b7-119">[値]</span><span class="sxs-lookup"><span data-stu-id="568b7-119">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="568b7-120">null 参照 (Visual Basic の場合は `Nothing`)。</span><span class="sxs-lookup"><span data-stu-id="568b7-120">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.NotFiniteNumberException.OffendingNumber%2A>|<span data-ttu-id="568b7-121">無効な数値。</span><span class="sxs-lookup"><span data-stu-id="568b7-121">The invalid number.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException(System::String ^ message);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="568b7-122">エラーを説明するメッセージ。</span><span class="sxs-lookup"><span data-stu-id="568b7-122">The message that describes the error.</span></span></param>
        <summary><span data-ttu-id="568b7-123">指定したエラー メッセージを使用して、<see cref="T:System.NotFiniteNumberException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="568b7-123">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with a specified error message.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="568b7-124">いることを確認、`message`文字列は、ユーザーが理解できるものと、現在のカルチャに合わせてローカライズされています。</span><span class="sxs-lookup"><span data-stu-id="568b7-124">Ensure that the `message` string is understandable to the user and that it is localized for the current culture.</span></span>  
  
 <span data-ttu-id="568b7-125"><xref:System.NotFiniteNumberException> のインスタンスの初期プロパティ値を次の表に示します。</span><span class="sxs-lookup"><span data-stu-id="568b7-125">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="568b7-126">プロパティ</span><span class="sxs-lookup"><span data-stu-id="568b7-126">Property</span></span>|<span data-ttu-id="568b7-127">[値]</span><span class="sxs-lookup"><span data-stu-id="568b7-127">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="568b7-128">null 参照 (Visual Basic の場合は `Nothing`)。</span><span class="sxs-lookup"><span data-stu-id="568b7-128">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="568b7-129">エラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="568b7-129">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected NotFiniteNumberException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; NotFiniteNumberException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info"><span data-ttu-id="568b7-130">シリアル化されたオブジェクト データを保持するオブジェクト。</span><span class="sxs-lookup"><span data-stu-id="568b7-130">The object that holds the serialized object data.</span></span></param>
        <param name="context"><span data-ttu-id="568b7-131">転送元または転送先に関するコンテキスト情報。</span><span class="sxs-lookup"><span data-stu-id="568b7-131">The contextual information about the source or destination.</span></span></param>
        <summary><span data-ttu-id="568b7-132">シリアル化したデータを使用して、<see cref="T:System.NotFiniteNumberException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="568b7-132">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with serialized data.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="568b7-133">このコンストラクターは、逆シリアル化中に、ストリームで送信された例外オブジェクトを再構築するために呼び出されます。</span><span class="sxs-lookup"><span data-stu-id="568b7-133">This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException (string message, double offendingNumber);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, float64 offendingNumber) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.String,System.Double)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, offendingNumber As Double)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException(System::String ^ message, double offendingNumber);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="offendingNumber" Type="System.Double" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="568b7-134">エラーを説明するメッセージ。</span><span class="sxs-lookup"><span data-stu-id="568b7-134">The message that describes the error.</span></span></param>
        <param name="offendingNumber"><span data-ttu-id="568b7-135">例外の原因となった引数の値。</span><span class="sxs-lookup"><span data-stu-id="568b7-135">The value of the argument that caused the exception.</span></span></param>
        <summary><span data-ttu-id="568b7-136">指定したエラー メッセージと、無効な数値を使用して、<see cref="T:System.NotFiniteNumberException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="568b7-136">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with a specified error message and the invalid number.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="568b7-137">いることを確認、`message`文字列は、ユーザーが理解できるものと、現在のカルチャに合わせてローカライズされています。</span><span class="sxs-lookup"><span data-stu-id="568b7-137">Ensure that the `message` string is understandable to the user and that it is localized for the current culture.</span></span>  
  
 <span data-ttu-id="568b7-138"><xref:System.NotFiniteNumberException> のインスタンスの初期プロパティ値を次の表に示します。</span><span class="sxs-lookup"><span data-stu-id="568b7-138">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="568b7-139">プロパティ</span><span class="sxs-lookup"><span data-stu-id="568b7-139">Property</span></span>|<span data-ttu-id="568b7-140">[値]</span><span class="sxs-lookup"><span data-stu-id="568b7-140">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="568b7-141">null 参照 (Visual Basic の場合は `Nothing`)。</span><span class="sxs-lookup"><span data-stu-id="568b7-141">A null reference (`Nothing` in Visual Basic).</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="568b7-142">エラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="568b7-142">The error message string.</span></span>|  
|<xref:System.NotFiniteNumberException.OffendingNumber%2A>|<span data-ttu-id="568b7-143">無効な数値。</span><span class="sxs-lookup"><span data-stu-id="568b7-143">The invalid number.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException(System::String ^ message, Exception ^ innerException);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="568b7-144">例外の原因を説明するエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="568b7-144">The error message that explains the reason for the exception.</span></span></param>
        <param name="innerException"><span data-ttu-id="568b7-145">現在の例外の原因となった例外。</span><span class="sxs-lookup"><span data-stu-id="568b7-145">The exception that is the cause of the current exception.</span></span> <span data-ttu-id="568b7-146"><c>innerException</c> パラメーターが null 参照 (Visual Basic の場合は <see langword="Nothing" />) でない場合は、内部例外を処理する <see langword="catch" /> ブロックで現在の例外が発生します。</span><span class="sxs-lookup"><span data-stu-id="568b7-146">If the <c>innerException</c> parameter is not a null reference (<see langword="Nothing" /> in Visual Basic), the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span></span></param>
        <summary><span data-ttu-id="568b7-147">指定したエラー メッセージと、この例外の根本的な原因である内部例外への参照を使用して、<see cref="T:System.NotFiniteNumberException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="568b7-147">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with a specified error message and a reference to the inner exception that is root cause of this exception.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="568b7-148">前の例外の直接の結果としてスローされる例外は、<xref:System.Exception.InnerException%2A> プロパティに前の例外への参照を含んでいる場合があります。</span><span class="sxs-lookup"><span data-stu-id="568b7-148">An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="568b7-149"><xref:System.Exception.InnerException%2A> プロパティは、コンストラクターに渡されたものと同じ値を返します。`null` プロパティによって内部例外値がコンストラクターに渡されなかった場合は、<xref:System.Exception.InnerException%2A> を返します。</span><span class="sxs-lookup"><span data-stu-id="568b7-149">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 <span data-ttu-id="568b7-150"><xref:System.NotFiniteNumberException> のインスタンスの初期プロパティ値を次の表に示します。</span><span class="sxs-lookup"><span data-stu-id="568b7-150">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="568b7-151">プロパティ</span><span class="sxs-lookup"><span data-stu-id="568b7-151">Property</span></span>|<span data-ttu-id="568b7-152">[値]</span><span class="sxs-lookup"><span data-stu-id="568b7-152">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="568b7-153">内部例外の参照。</span><span class="sxs-lookup"><span data-stu-id="568b7-153">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="568b7-154">エラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="568b7-154">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public NotFiniteNumberException (string message, double offendingNumber, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, float64 offendingNumber, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.#ctor(System.String,System.Double,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, offendingNumber As Double, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; NotFiniteNumberException(System::String ^ message, double offendingNumber, Exception ^ innerException);" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="offendingNumber" Type="System.Double" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message"><span data-ttu-id="568b7-155">例外の原因を説明するエラー メッセージ。</span><span class="sxs-lookup"><span data-stu-id="568b7-155">The error message that explains the reason for the exception.</span></span></param>
        <param name="offendingNumber"><span data-ttu-id="568b7-156">例外の原因となった引数の値。</span><span class="sxs-lookup"><span data-stu-id="568b7-156">The value of the argument that caused the exception.</span></span></param>
        <param name="innerException"><span data-ttu-id="568b7-157">現在の例外の原因となった例外。</span><span class="sxs-lookup"><span data-stu-id="568b7-157">The exception that is the cause of the current exception.</span></span> <span data-ttu-id="568b7-158"><c>innerException</c> パラメーターが null 参照 (Visual Basic の場合は <see langword="Nothing" />) でない場合は、内部例外を処理する <see langword="catch" /> ブロックで現在の例外が発生します。</span><span class="sxs-lookup"><span data-stu-id="568b7-158">If the <c>innerException</c> parameter is not a null reference (<see langword="Nothing" /> in Visual Basic), the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span></span></param>
        <summary><span data-ttu-id="568b7-159">指定したエラー メッセージ、無効な数値、およびこの例外の原因である内部例外への参照を使用して、<see cref="T:System.NotFiniteNumberException" /> クラスの新しいインスタンスを初期化します。</span><span class="sxs-lookup"><span data-stu-id="568b7-159">Initializes a new instance of the <see cref="T:System.NotFiniteNumberException" /> class with a specified error message, the invalid number, and a reference to the inner exception that is root cause of this exception.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="568b7-160">前の例外の直接の結果としてスローされる例外は、<xref:System.Exception.InnerException%2A> プロパティに前の例外への参照を含んでいる場合があります。</span><span class="sxs-lookup"><span data-stu-id="568b7-160">An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="568b7-161"><xref:System.Exception.InnerException%2A> プロパティは、コンストラクターに渡されたものと同じ値を返します。`null` プロパティによって内部例外値がコンストラクターに渡されなかった場合は、<xref:System.Exception.InnerException%2A> を返します。</span><span class="sxs-lookup"><span data-stu-id="568b7-161">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or `null` if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.</span></span>  
  
 <span data-ttu-id="568b7-162"><xref:System.NotFiniteNumberException> のインスタンスの初期プロパティ値を次の表に示します。</span><span class="sxs-lookup"><span data-stu-id="568b7-162">The following table shows the initial property values for an instance of <xref:System.NotFiniteNumberException>.</span></span>  
  
|<span data-ttu-id="568b7-163">プロパティ</span><span class="sxs-lookup"><span data-stu-id="568b7-163">Property</span></span>|<span data-ttu-id="568b7-164">[値]</span><span class="sxs-lookup"><span data-stu-id="568b7-164">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="568b7-165">内部例外の参照。</span><span class="sxs-lookup"><span data-stu-id="568b7-165">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="568b7-166">エラー メッセージ文字列。</span><span class="sxs-lookup"><span data-stu-id="568b7-166">The error message string.</span></span>|  
|<xref:System.NotFiniteNumberException.OffendingNumber%2A>|<span data-ttu-id="568b7-167">無効な数値。</span><span class="sxs-lookup"><span data-stu-id="568b7-167">The invalid number.</span></span>|  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Exception" />
      </Docs>
    </Member>
    <Member MemberName="GetObjectData">
      <MemberSignature Language="C#" Value="public override void GetObjectData (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig virtual instance void GetObjectData(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.NotFiniteNumberException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Public Overrides Sub GetObjectData (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; override void GetObjectData(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberType>Method</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <Attributes>
        <Attribute>
          <AttributeName>System.Security.SecurityCritical</AttributeName>
        </Attribute>
      </Attributes>
      <ReturnValue>
        <ReturnType>System.Void</ReturnType>
      </ReturnValue>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info"><span data-ttu-id="568b7-168">シリアル化されたオブジェクト データを保持するオブジェクト。</span><span class="sxs-lookup"><span data-stu-id="568b7-168">The object that holds the serialized object data.</span></span></param>
        <param name="context"><span data-ttu-id="568b7-169">転送元または転送先に関するコンテキスト情報。</span><span class="sxs-lookup"><span data-stu-id="568b7-169">The contextual information about the source or destination.</span></span></param>
        <summary><span data-ttu-id="568b7-170"><see cref="T:System.Runtime.Serialization.SerializationInfo" /> オブジェクトに無効な数値と追加例外情報を設定します。</span><span class="sxs-lookup"><span data-stu-id="568b7-170">Sets the <see cref="T:System.Runtime.Serialization.SerializationInfo" /> object with the invalid number and additional exception information.</span></span></summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="568b7-171"><xref:System.TypeLoadException.GetObjectData%2A> は、シリアル化する対象のすべての例外オブジェクト データを使用して、<xref:System.Runtime.Serialization.SerializationInfo> を設定します。</span><span class="sxs-lookup"><span data-stu-id="568b7-171"><xref:System.TypeLoadException.GetObjectData%2A> sets a <xref:System.Runtime.Serialization.SerializationInfo> with all the exception object data targeted for serialization.</span></span> <span data-ttu-id="568b7-172">逆シリアル化中に例外オブジェクトを再構成、<xref:System.Runtime.Serialization.SerializationInfo>ストリームで転送します。</span><span class="sxs-lookup"><span data-stu-id="568b7-172">During deserialization, the exception object is reconstituted from the <xref:System.Runtime.Serialization.SerializationInfo> transmitted over the stream.</span></span>  
  
 <span data-ttu-id="568b7-173">詳細については、「<xref:System.Runtime.Serialization.SerializationInfo>」を参照してください。</span><span class="sxs-lookup"><span data-stu-id="568b7-173">For more information, see <xref:System.Runtime.Serialization.SerializationInfo>.</span></span>  
  
 ]]></format>
        </remarks>
        <exception cref="T:System.ArgumentNullException"><span data-ttu-id="568b7-174"><paramref name="info" /> オブジェクトが <see langword="null" /> です。</span><span class="sxs-lookup"><span data-stu-id="568b7-174">The <paramref name="info" /> object is <see langword="null" />.</span></span></exception>
        <permission cref="T:System.Security.SecurityCriticalAttribute"><span data-ttu-id="568b7-175">直前の呼び出し元に対する完全な信頼が必要です。</span><span class="sxs-lookup"><span data-stu-id="568b7-175">requires full trust for the immediate caller.</span></span> <span data-ttu-id="568b7-176">このメンバーは、部分的に信頼されているまたは透過的なコードで使用することはできません。</span><span class="sxs-lookup"><span data-stu-id="568b7-176">This member cannot be used by partially trusted or transparent code.</span></span></permission>
        <altmember cref="T:System.Runtime.Serialization.SerializationInfo" />
        <altmember cref="T:System.Runtime.Serialization.StreamingContext" />
      </Docs>
    </Member>
    <Member MemberName="OffendingNumber">
      <MemberSignature Language="C#" Value="public double OffendingNumber { get; }" />
      <MemberSignature Language="ILAsm" Value=".property instance float64 OffendingNumber" />
      <MemberSignature Language="DocId" Value="P:System.NotFiniteNumberException.OffendingNumber" />
      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property OffendingNumber As Double" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; property double OffendingNumber { double get(); };" />
      <MemberType>Property</MemberType>
      <AssemblyInfo>
        <AssemblyName>mscorlib</AssemblyName>
        <AssemblyVersion>2.0.5.0</AssemblyVersion>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>netstandard</AssemblyName>
        <AssemblyVersion>2.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <AssemblyInfo>
        <AssemblyName>System.Runtime</AssemblyName>
        <AssemblyVersion>4.2.0.0</AssemblyVersion>
        <AssemblyVersion>4.2.1.0</AssemblyVersion>
      </AssemblyInfo>
      <ReturnValue>
        <ReturnType>System.Double</ReturnType>
      </ReturnValue>
      <Docs>
        <summary><span data-ttu-id="568b7-177">正の無限大、負の無限大、または非数 (NaN) である無効な数値を取得します。</span><span class="sxs-lookup"><span data-stu-id="568b7-177">Gets the invalid number that is a positive infinity, a negative infinity, or Not-a-Number (NaN).</span></span></summary>
        <value><span data-ttu-id="568b7-178">無効な数値。</span><span class="sxs-lookup"><span data-stu-id="568b7-178">The invalid number.</span></span></value>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="568b7-179"><xref:System.NotFiniteNumberException.OffendingNumber%2A>プロパティは、コンス トラクターに渡されたものと同じ値を返しますまたは、ない場合は 0 の値が指定されました。</span><span class="sxs-lookup"><span data-stu-id="568b7-179">The <xref:System.NotFiniteNumberException.OffendingNumber%2A> property returns the same value as was passed into the constructor, or zero if no value was supplied.</span></span> <span data-ttu-id="568b7-180">上書き、<xref:System.NotFiniteNumberException.OffendingNumber%2A>コンテンツまたは、無効な値の書式をカスタマイズするプロパティです。</span><span class="sxs-lookup"><span data-stu-id="568b7-180">Override the <xref:System.NotFiniteNumberException.OffendingNumber%2A> property to customize the content or format of the invalid value.</span></span>  
  
 <span data-ttu-id="568b7-181">このプロパティは読み取り専用です。</span><span class="sxs-lookup"><span data-stu-id="568b7-181">This property is read-only.</span></span>  
  
 ]]></format>
        </remarks>
        <altmember cref="T:System.Double" />
      </Docs>
    </Member>
  </Members>
</Type>